pyproject.toml

```diff
@@ -71,28 +62,30 @@ doc = [
     "sphinx-design",
     "sphinx-gallery",
     "sphinxcontrib-bibtex",
-    "sphinxext-opengraph"
+    "sphinxext-opengraph",
+    "ruamel.yaml"
 ]
-# For testing of oldest usable versions of dependencies.
-min = [
-    "joblib==1.0.0",
-    "nibabel==3.2.0",
-    "numpy==1.19.0",
-    "pandas==1.1.5",
-    "scikit-learn==1.0.0",
-    "scipy==1.6.0"
-]
-# For surface plotting mostly
+# Necessary req to use nilearn's plotting module
 # Kaleido version is pinned for windows due to bug
 # See https://github.com/plotly/Kaleido/issues/134 for more detail
-plotly = [
+plotting = [
```

Comment:
Was just thinking that changing the name of that list of optional dependencies may "break" some users workflow.

Maybe better to have an "alias" like:

```
plotly = ["nilearn[plotting]"]
```

Though not technically API, not sure if we should not throw some sort of deprecation warning

Reply:
In any case this sounds like it is worth mentioning in the changelog

Reply:
or maybe just keep the original "plotting" and "plotly" original names and content...

Reply:
OK: sounds reasonable

I will still add the option: `plotly = ["nilearn[plotting]"]` because this won't lead to any breakage.

The 'only' behavioral change this will do is that the `plotting` option will now also install plotly and kaleido.